### ISSUE TITLE
Can't remove "placeholder" from datetime alias

### ISSUE THREAD

**Issue by rafaelss95** [RobinHerbots/Inputmask#2438]
- Describe the bug

```html
Input format (dd/MM/yyyy): <input type="text" class="test-day-month-year">
```

```js
$('.test-day-month-year').inputmask({ 
  alias: 'datetime', inputFormat: 'dd/MM/yyyy', jitMasking: true, placeholder: ''
});
```

When you start typing, it includes a "M" in the month part even if I specify `jitMasking: true` and `placeholder: ''` (even if I'm not sure placeholder is needed here), but that doesn't happen if you have another format like `MM/yyyy`, for example.

![Captura de Tela 2020-12-09 às 22 27 11](https://user-images.githubusercontent.com/11965907/101709214-d5ef2d00-3a6d-11eb-97bf-4754ab2d2a29.png)

- Add a link to a codepen, jsfiddle or other example page which shows the problem
https://jsfiddle.net/cp4dkqas/

- OS:
Tested on macOS Catalina 10.15.4.

- Browser
Tested on Chrome 87 and Safari 13.

- Inputmask version
5.0.5.

**Comment by RobinHerbots** [comment]
Hi @rafaelss95 ,

Besides the bug.  You should use lowercase mm for month. MM = minutes.
I will see why the placeholder is appended.

**Comment by RobinHerbots** [comment]
@rafaelss95 , 

The issue is fixed in version 5.0.6-beta.25
